packages/design-system/src/components/UsaBanner/UsaBanner.jsx

```diff
@@ -85,6 +94,11 @@ UsaBanner.defaultProps = {
 
 UsaBanner.propTypes = {
   /**
+   * A unique `id` to be used on the usa banner.
```

Comment:
`A unique ID to be applied to the banner content. A unique ID will be generated if one isn't provided.`